configure.ml

```diff
@@ -13,7 +13,7 @@ open Printf
 
 let coq_version = "8.8+alpha"
```

Comment:
Is this variable still used anywhere? My impression is that it isn't, in particular because of the fragile approach in case we are not in a git repository (see other comment).

Reply:
Now used when git is not found, and no `coqversion.ml` file has been provided.

Reply:
It seems poor that this appears both in `configure.ml` and `dev/gen_version.sh`...